wolfcrypt/src/misc.c

```diff
@@ -183,9 +183,18 @@ STATIC INLINE void xorbuf(void* buf, const void* mask, word32 count)
 /* Make sure compiler doesn't skip */
 STATIC INLINE void ForceZero(const void* mem, word32 len)
 {
+#ifdef WOLFCRYPT_FORCEZERO_BYTE
     volatile byte* z = (volatile byte*)mem;
 
     while (len--) *z++ = 0;
+#else
+    volatile word64 *z = (volatile word64 *)mem;
```

Comment:
We can't assume word64 is going to be available.  On some platforms it isn't.  Of course if an algorithm needs it, like SHA512 then we do assume (but only there).  Also, some platforms will fault if mem isn't aligned here when it's dereferenced.